src/interfaces/IContactUs.ts

```diff
@@ -0,0 +1,22 @@
+//Contact Us Interface
+interface IContactUs {
+  _id?: string;
+  name: string;
+  email: string;
+  message: string;
+  createdAt: Date;
+  updatedAt: Date;
+  deletedAt?: Date | null;
+}
+
+//Contact Us State interface
+interface IContactUsStore {
```

Comment:
Add selectedContactUsId and addContactUs too

Reply:
addContactUs is not required in admin panel @miyurugunarathna @ThaminduChankana 
It is handling in our web

Reply:
Ahh yes yes.